### ISSUE TITLE
[HIPIFY][perl] hipify-perl should support --help and --version options

### ISSUE THREAD

**Issue by boegel** [ROCm-Developer-Tools/HIPIFY#290]
It would be useful if `hipify-perl` would support a `--help` option, to give information about basic usage, and to print all supported options (like `--inplace`).

```
$ hipify-perl --help
Unknown option: help
```

Likewise, supporting `--version` would be helpful too.

**Comment by emankov** [comment]
What do you want from the script under the `--version` option?

**Comment by boegel** [comment]
@emankov Well, just print the version number? Something like:

```
$ hipify-perl --version
4.0.0
```

**Comment by emankov** [comment]
So, do you need ROCm version?

**Comment by boegel** [comment]
I don't need anything in particular, I mostly would like that `--version` is supported, since it's a pretty common option for CLI tools.
If HIPIFY doesn't have it's own version number, then printing the ROCm version makes sense I guess.

**Comment by emankov** [comment]
`hipify-clang` being clang tooling based tool prints LLVM version against which it is built. For `hipify-perl` let it be the HIP version.

**Comment by emankov** [comment]
Closing it as implemented by #291 and #292.
